C/plugins/storage/sqlite/common/readings.cpp

```diff
@@ -2246,9 +2246,9 @@ vector<string>  assetCodes;
 
 		totTime += usecs;
 
-		if(usecs>150000)
+		if(usecs > 150000)
 		{
-			std::this_thread::sleep_for(std::chrono::milliseconds(100+usecs/10000));
+			std::this_thread::sleep_for(std::chrono::milliseconds(100 + usecs/1000));
```

Comment:
@MarkRiddoch : Division by 10000 seemed like an error, please cross check.

Reply:
Yes, it probably was